service/source.go

```diff
@@ -385,15 +385,37 @@ func (g *AutoParameterGenerator) Validate(name string, defaultValue interface{},
 	return err
 }
 
-func (g *AutoParameterGenerator) split(tag string) (source string, name string, err error) {
+func (g *AutoParameterGenerator) split(tag string) (source, name, defaultValue string, err error) {
 	result := strings.Split(tag, ",")
-	if len(result) == 1 {
-		return result[0], "", nil
-	}
-	if len(result) != 2 {
-		return "", "", invalidFieldTag.Error(tag)
+
+	switch len(result) {
+	case 1:
+		source = strings.TrimSpace(result[0])
+	case 2:
+		source = strings.TrimSpace(result[0])
+		name = strings.TrimSpace(result[1])
+	case 3:
+		source = strings.TrimSpace(result[0])
+		name = strings.TrimSpace(result[1])
+		params := strings.Split(strings.TrimSpace(result[2]), ";")
```

Comment:
`path,application,default=123,anotherKey=anotherValue` is more preferable than `Path,application,default=123;anotherKey=anotherValue`.